<ui version="4.0" >
 <class>KWin::MagnifierEffectConfigForm</class>
 <widget class="QWidget" name="KWin::MagnifierEffectConfigForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupSize" >
     <property name="title" >
      <string>Size</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>&amp;Width:</string>
        </property>
        <property name="buddy" >
         <cstring>spinWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="spinWidth" >
        <property name="suffix" >
         <string> px</string>
        </property>
        <property name="maximum" >
         <number>500</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>&amp;Height:</string>
        </property>
        <property name="buddy" >
         <cstring>spinHeight</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="spinHeight" >
        <property name="suffix" >
         <string> px</string>
        </property>
        <property name="maximum" >
         <number>500</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KWin::GlobalShortcutsEditor" native="1" name="editor" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KWin::GlobalShortcutsEditor</class>
   <extends>QWidget</extends>
   <header location="global" >kwineffects.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
